be/src/exec/spill/input_stream.h

```diff
@@ -0,0 +1,74 @@
+// Copyright 2021-present StarRocks, Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#pragma once
+
+#include <atomic>
+
+#include "column/vectorized_fwd.h"
+#include "common/statusor.h"
+#include "exec/sort_exec_exprs.h"
+#include "exec/sorting/sorting.h"
+#include "exec/spill/block_manager.h"
+#include "exec/spill/formatter.h"
+
+namespace starrocks {
+namespace spill {
+
+// InputStream is used in restore phase to represent an input stream of a restore task.
+// InputStream reads multiple Blocks and returns the deserialized Chunks.
+class InputStream {
```

Comment:
we prefer a spec name eg: SpilledInputStream

Reply:
> we prefer a spec name eg: SpilledInputStream

we have put InputStream into `spill` namespace, it needn't add `Spilled` prefix